enclave_manager/avalon_enclave_manager/wpe/enclave_manager.py

```diff
@@ -0,0 +1,379 @@
+#!/usr/bin/env python3
```

Comment:
I would suggest to rename this file to wpe_encalve_manager.py to avoid any ambiguity. Else if we search for this file in repo we get many versions with same name.